Merit_Visualization.js

```diff
@@ -0,0 +1,304 @@
+//////////////////////////////////////////////////////////////
+// Asset List
+//////////////////////////////////////////////////////////////
+
+var merit = ee.Image('users/meritdataset/Merit-Hydro-world');
+var elv = merit.select('elv');
+var flowDirection = merit.select('dir');
+var riverwidth = merit.select('wth');
+var waterArea = merit.select('wat');
+var upArea = merit.select('upa');
+var hand = merit.select('hnd');
+var viswidth = merit.select('viswth');
+
+var merit_rd = ee.Image('users/meritdataset/Merit-Hydro-world_reduced');
```

Comment:
this asset is not in the catalog - should we add it?

Reply:
I understand. Then is it possible to include the asset in the catalog?